File: Sigmath/Sigmath/Sigmath.ConvertedToRuby/calculus/matrix.rb
require "mscorlib"
require "System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

module Sigmath
	class Matrix
		def Rows
		end

		def Cols
		end

		def initialize(data)
			self.Rows = data.GetLength(0)
			self.Cols = data.GetLength(1)
			@_data = data
		end

		def initialize(data)
			self.Rows = data.GetLength(0)
			self.Cols = data.GetLength(1)
			@_data = data
		end

		def Item
		end
	end
end

File: Sigmath.ConvertedToPython/calculus/matrix.py
from System import *

class Matrix(object):
	def get_Rows(self):

	Rows = property(fget=get_Rows)

	def get_Cols(self):

	Cols = property(fget=get_Cols)

	def __init__(self, data):
		self.Rows = data.GetLength(0)
		self.Cols = data.GetLength(1)
		self.__data = data

	def __init__(self, data):
		self.Rows = data.GetLength(0)
		self.Cols = data.GetLength(1)
		self.__data = data

	def get_Item(self):

	Item = property(fget=get_Item)